assets/queries/k8s/cronjob_deadline_not_configured/query.rego

```diff
@@ -0,0 +1,16 @@
+package Cx
+
+CxPolicy [ result ] {
+  spec := input.document[i].spec
```

Comment:
Add metadata information to improve searchKey:

document := input.document[i]
spec := document .spec
metadata := document.metadata

Reply:
Done